docs/sources/tempo/configuration/_index.md

```diff
@@ -707,6 +710,34 @@ query_frontend:
 
 ```
 
+### Limit query size to improve performance and stability
+
+Querying large tracing data presents several challenges.
+Span sets with large number of spans impact query performance and stability.
+In a similar manner, excessive queries result size can also negatively impact query performance.
+
+#### Limit the spans per spanset
+
+You can set the maximum spans per spanset by setting `max_spans_per_span_set` for the query-frontend.
+The default value is 100. Tempo and Grafana Enterprise Traces users can set this option in their configuration file.
```

Comment:
We don't need to say that the option can be set in the config file since we're in the config file doc. 
```suggestion
The default value is 100. 
```